src/main/java/com/minelittlepony/PonyRace.java

```diff
@@ -8,7 +8,11 @@
     UNICORN(false, true),
     ALICORN(true, true),
     CHANGELING(true, true),
-    ZEBRA(false, false);
+    ZEBRA(false, false),
+    REFORMED_CHANGELING(true, true),
```

Comment:
Changedling

Reply:
And what is a changedling but a recolored changeling?

Reply:
Similar to how a zebra is a recolored earth pony?

Reply:
To be honest, this is all a bit of legacy code. Truly, there's only 4 races, and they are just body types. See the below table.

||Wings|No Wings|
|--|-----------|--------------|
|**Horn**|Alicorn|Unicorn|
|**No Horn**|Pegasus|Earth Pony|

Reply:
And the other two were added in version 1.4.6.3 "to support mods that want to add abilities to zebras and changlings (they just use the earth pony and alicorn models respectively)." Still only the 4 model types/races, like you're saying.
I'm currently just offering this up while I play in the code for pony wing rendering so i can change the sizes of the wings.

Reply:
That's fair enough. Just know a few things.

1. I don't like the way this system works. Expect it to change in the future.
2. This probably won't get released for 1.12. 1.13 at the soonest.

Reply:
Should I hold off on my wing sizing code until you've redesigned things?

Reply:
@killjoy1221 Do you have any plans in terms of changing the system? Do they affect the skin format or just code logic? Curious from the perspective of a MineLP addon developer.

Reply:
I don't plan on doing it anytime soon, no. I just want to separate the race section from determining what model you are. Does that make sense?

Reply:
In the code? Up to you. In the skin fomat? No - I don't think, say, an earth pony who defines themselves to be a pegasus by race but has an earth pony model should be allowed to fly. Models should still remain tied to races, I think.

Reply:
I don't plan on changing the format. The current format should always work with all future versions.

Reply:
My mods, which added the functional part of each pony race, have usually decided which race's qualities should be applied based on the data in the skin itself, as to avoid defining it twice and to make it easier for players. This is, effectively, the functionality I wouldn't want to be broken.

Reply:
I'll try to work around that when I do find time to work on this. Though it's not likely I ever will.

Reply:
I'd look into this myself, but I'm not a LiteLoader developer - I'm a Forge developer. That complicates things slightly.

Reply:
The only parts of liteloader I use is configs, mixins and some event hooks. You can get by if you just know the vanilla codebase.

Reply:
Hmm. In that case, wouldn't it be possible to modify the mod so it can use both Forge+Mixin and LiteLoader+Mixin as backends, that is have both a Forge and LiteLoader version? That could probably work for me, making it much easier to, say, test the mod adequately in Forge environments (like for my cosmetics mod PonySocks).